src/core/streaming_playback_manager.py

```
@@ -843,6 +845,16 @@ async def _stream_audio_loop(
                         except Exception:
                             pass
+                        
+                        if bytes_since_last_upsert > 0:
+                            try:
+                                sess = await self.session_store.get_by_call_id(call_id)
+                                if sess:
+                                    sess.streaming_bytes_sent += bytes_since_last_upsert
+                                    sess.streaming_jitter_buffer_depth = jitter_buffer.qsize()
+                                    await self.session_store.upsert_call(sess)
+                            except Exception:
+                                pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)